components/airtable/new-records-in-view.js

```diff
@@ -31,7 +31,7 @@ module.exports = {
 
     let maxTimestamp
     const lastMaxTimestamp = this.db.get("lastMaxTimestamp")
-    if (!lastMaxTimestamp) {
+    if (lastMaxTimestamp) {
```

Comment:
was the `!lastMaxTimestamp` a prior bug you fixed here?

Reply:
yes